### PR TITLE
OMC compute example

### PR DESCRIPTION
**Pull request by jimmycal**
I have added an example under compute to show how an instance can be created that is managed by our Oracle Management Cloud.  The readme still needs some work, but I wanted your input on the structure, setup of the example.

### CODE REVIEW

**Comment by josephholsten** [comment]
should we close this ticket? looks like the changes have been merged elsewhere

**Comment by jvcalise** [comment]
No I need to correct the commit on the branch. I will update it shortly.

File: docs/examples/compute/omc_managed_server/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    #    \___/|_| \_/_/   \_\____|_____|_____|
+***
+## Bare Metal Cloud Instance managed with Oracle Management Cloud Agent
+These two configurations 
+
+It creates a VN
+
+### Using this example
+* This example is predicated on your usage of Oracle Management Cloud, and having downloaded the AgentInstall.zip and generating an Agent Registration Key
+Details can be found here - <some link>
```
Comment (josephholsten) [inline docs/examples/compute/omc_managed_server/README.md]: Still needs that link

File: docs/examples/compute/omc_managed_server/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+* Update env-vars with the required information. Most examples use the same set of environment variables so you only need to do this once.
+* Source env-vars
+  * `$ . env-vars`
+
+* Two paths are available to create an OMC managed instance. 1. The reccomended approach is a two stage deploy, where we first load the agent, and download necessary agent files and create and image that we can reuse. This approach allows you to do the time consuming work one time, and create an image to be used to configure the agent on demand.  2. Install and configure the agent in one single step.  This approach takes about 20 minutes to bring a server online.
+
+### Two  Stage Deploy
```
Comment (josephholsten) [inline docs/examples/compute/omc_managed_server/README.md]: double spaces between words

File: docs/examples/compute/omc_managed_server/README.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    * The compartment name to run your instance
+    * The OCID for the subnet where you want your instance to run.
+        * It is required that this subnet be public and allow SSH traffic
+    * The path to your AgentInstall.zip
+    * Your Agent registration key
+* Load the resource module 
+* `$ terraform get`
+* `$ terraform plan -var-file=example.tfvars`
+* `$ terraform apply -var-file=example.tfvars`
+* Once your instance is running, goto the console, and create a custom image from this server.  Note the ocid of the image that is created.
```
Comment (josephholsten) [inline docs/examples/compute/omc_managed_server/README.md]: if only implementing a packer provider and a terraform provider were the same task 😿

File: docs/examples/compute/omc_managed_server/create_single_omc_server/omc_config/install_omc.tpl [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+cat /omc/stage/omc_entity.json | jq '.entities[0].name="'$(hostname -f)'"' | cat > /omc/stage/omc_entity_update.json
```
Comment (josephholsten) [inline docs/examples/compute/omc_managed_server/create_single_omc_server/omc_config/install_omc.tpl]: Maybe a comment about why this script exists and how it works?

**Comment by jvcalise** [comment]
Should be good to go

**Comment by jimmycal** [comment]
Pushed an update with the logs removed

**Pull request merged**